src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1012,6 +1065,11 @@ type StopGeometries {
     googleEncoded: [Geometry]
 }
 
+input Coordinate {
```

Comment:
I thought it was considered bad practice in GraphQL to repeat the "type" of the entry in the name. I havent' checked if it's possible to give the same name to two entries of different types but I can see how that could be slightly confusing.

Reply:
I rechecked this and I think you are right. Having the input suffix seems to be somewhat standard.

Reply:
I somehow managed to mess this up while renaming and used prefix instead of suffix. In the schema we already had some types that used prefix. Should we use suffix or prefix in new input types?